### PR TITLE
Anna/check version

### PR DESCRIPTION
**Pull request by Hoshiyo**

### CODE REVIEW

File: fabfile/tasks.py [hunk 1]
```diff
@@ -130,6 +132,14 @@ def upgrade_all(bina=True, up_tyr=True, up_confs=True, kraken_wait=True):
             execute(upgrade_jormungandr, up_confs=up_confs)
 
 @task
+def compare_version_candidate_installed(no_load):
+    """Check candidate version is different from installed"""
+    message = "Candidate kraken version is older or the same than the installed one."
+    if not execute(show_version, action='check') and no_load:
+        print(red(message))
+        exit(1)
```
Comment (xlqian) [inline fabfile/tasks.py]: exit() or throw exception?

Comment (francois-vincent) [inline fabfile/tasks.py]: no_reload : use a more specific name : eg check_version

Comment (francois-vincent) [inline fabfile/tasks.py]: function compare_version_candidate_installed() should have no parameter

File: fabfile/tasks.py [hunk 2]
```diff
@@ -92,8 +93,9 @@ def upgrade_all_packages():
     execute(jormungandr.upgrade_ws_packages)
 
 @task
-def upgrade_all(bina=True, up_tyr=True, up_confs=True, kraken_wait=True):
+def upgrade_all(bina=True, up_tyr=True, up_confs=True, kraken_wait=True, no_reload=True):
```
Comment (francois-vincent) [inline fabfile/tasks.py]: no_reload : use a more specific name : eg check_version

File: fabfile/tasks.py [hunk 3]
```diff
@@ -130,6 +132,14 @@ def upgrade_all(bina=True, up_tyr=True, up_confs=True, kraken_wait=True):
             execute(upgrade_jormungandr, up_confs=up_confs)
 
 @task
+def compare_version_candidate_installed(no_load):
```
Comment (francois-vincent) [inline fabfile/tasks.py]: function compare_version_candidate_installed() should have no parameter

File: fabfile/utils.py [hunk 4]
```diff
@@ -268,7 +269,11 @@ def summarize(iterable):
         candidate = summarize(x[1] for x in versions.itervalues())
         if isinstance(installed, tuple):
             installed = max(installed)
-        return installed != candidate and candidate
+
+        if semver.compare(candidate,installed) >= 0:
```
Comment (francois-vincent) [inline fabfile/utils.py]: operator should be '>', not '>='

File: fabfile/tasks.py (hunk 1 above)
Comment (francois-vincent) [inline fabfile/tasks.py]: fabric.api.abort

**Pull request merged**